core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.directive.html

```diff
@@ -2,7 +2,7 @@
   <div>
     <div class="oppia-exp-interaction-card-header-container oppia-mobile-collapsible-card-header" ng-click="toggleInteractionEditor()">
       <h3 class="oppia-exp-interaction-card-header">
-        Interaction
+        Interaction {{getCurrentInteractionName().header}}
```

Comment:
Have you checked for all interactions and made sure they in a human readable form (i.e. they don't use backend names like RatioInput)?

Reply:
Yes, whatever names are present in the choose interaction modal are displayed here. Check the description.

Reply:
awesome, thanks!